middleware.ts

```
@@ -0,0 +1,35 @@
+const publicRoutes = [
+  "/",
+  "/sign-in(.*)",
+  "/sign-up(.*)",
+  "/api/webhooks(.*)",
+  "/accessibility-test(.*)",
+  "/responsive-demo(.*)",
+  "/theme-demo(.*)",
+];
+
+// Routes to be ignored by the middleware completely
+const ignoredRoutes = [
```

ESLint

[error] 'ignoredRoutes' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)